stratum/lib/security/README.md

```diff
@@ -0,0 +1,35 @@
+Stratum securoty components
+====
```

Comment:
I think we mainly use the `#` style for headings.
# `#` for title
## `##` for 1. level headings